pyproject.toml

```diff
@@ -72,7 +73,11 @@ extend-exclude = [
 
 [tool.ruff.per-file-ignores]
 "dev/*" = ["T201", "PT018"]
-"mlflow/*" = ["PT018"]
+"mlflow/*" = [
+  "PT018",
+  # TODO: Sort imports in files under `mlflow` as well
+  "I",
+]
```

Comment:
Disabled for now because changing the import order may lead to circular imports.